sycl/include/sycl/aliases.hpp

```diff
@@ -73,6 +90,9 @@ using ulong = unsigned long;
 using longlong = long long;
 using ulonglong = unsigned long long;
 using half = sycl::detail::half_impl::half;
+
+// FIXME: SYCL 2020 spec says that cl_* aliases should reside in sycl::opencl
+// namespace.
```

Comment:
What is blocking us from applying these FIXME comments right away?

Reply:
Apparently, we still use some of these aliases: #7888. I don't know exactly about `cl_*` scalar aliases, but we for sure use `cl_*` vector aliases.

I would prefer an incremental approach and get this merged first as a fix for vector aliases and then submit a separate patch for scalar aliases

Reply:
I took another look at this: vector `cl_*` aliases depend on scalar `cl_*` aliases and those in turn are still used (#7888)

Reply:
> Apparently, we still use some of these aliases: #7888. I don't know exactly about `cl_*` scalar aliases, but we for sure use `cl_*` vector aliases.
> 
> I would prefer an incremental approach and get this merged first as a fix for vector aliases and then submit a separate patch for scalar aliases

I also prefer incremental approach, which should be remove `cl_*` alias usages from DPC++ code base first and remove alias definitions after that. This way there will be no technical debt.